src/luacov/runner.lua

```diff
@@ -397,6 +402,21 @@ local function has_hook_per_thread()
 end
 
 --------------------------------------------------
+-- Wraps a function to be used in a coroutine created via C API.
```

Comment:
If I follow correctly, this exists only because of LuaJIT? If so, I think it should be documented. (If not, the use cases explaining why this function exists should be mentioned.)